chain/epoch_manager/src/lib.rs

```diff
@@ -71,12 +71,15 @@ pub struct EpochManager {
     epoch_info_aggregator: Option<EpochInfoAggregator>,
     /// Largest final height. Monotonically increasing.
     largest_final_height: BlockHeight,
+    /// Number of additional epochs to keep in memory.
```

Comment:
Can you add a comment saying that it is additional to `NUM_EPOCHS_TO_KEEP_STORE_DATA` epochs?

Reply:
Actually, could we allow a lower number? I think it is technically possible to only the past 2 epochs of data